channeldb/payments.go

```diff
@@ -92,6 +92,14 @@ var (
 	// paymentFailInfoKey is a key used in the payment's sub-bucket to
 	// store information about the reason a payment failed.
 	paymentFailInfoKey = []byte("payment-fail-info")
+
+	// paymentsIndexBucket is the name of the top-level bucket within the
+	// database that stores an index of payment sequence numbers to its
+	// payment hash.
+	// payments-sequence-index-bucket
+	// 	|--sequence-number: <payment hash>
+	//      |--sequence-number: <payment hash>
```

Comment:
two identical keys?

Reply:
Ah, wasn't clear on the schema for these comments. Wrapped in `<>` to indicate that they're keyed by value, not a string.